res/controllers/Numark_DJ2GO2_Touch_scripts.js

```diff
@@ -104,7 +104,14 @@ DJ2GO2Touch.Deck = function(deckNumbers, midiChannel) {
         this.hotcueButtons[i] = new components.HotcueButton({
             group: "[Channel" + script.deckFromGroup(this.currentDeck) + "]",
             midi: [0x94 + midiChannel, 0x00 + i],
-            number: i
+            number: i,
+            input: function(channel, control, value, status, _group) {
+                if (control <= 8) {
```

Comment:
From your comment above
```suggestion 
                // DJ2GO2/DJ2GO2 Touch does not have a "shift" button, 
                // but when in CUES Pad Mode, holding the PAD MODE button will 
                // cause the controller to send messages where the originating midi
                // control is offset by 8.
                // this "feature" only applies to the CUES PAD MODE. Holding the button
                // in other modes does nothing.
                if (control <= 8) {
```